.travis.yml

```diff
@@ -13,29 +15,29 @@ env:
     - GIT_CI_USER: TravisCI
     - GIT_CI_EMAIL: TravisCI@mdanalysis.org
     - MDA_DOCDIR: package/doc/html
+  matrix:
```

Comment:
Yeah that is a very good chance. I'm not aware of duplicate keys in the YAML standard.